### DISCUSSION TITLE
Icons are not showing and cant travel through the site

### DISCUSSION THREAD

**Discussion by VasuTech1** [kimai/kimai#4347] [On-Premise: Installation & Updates]
Hello Guys,
![Kimai error](https://github.com/kimai/kimai/assets/109790703/7338b426-c64b-4308-9d67-b2200b5b0236)

I have recently installed Kimai version 2.0.35 on production environment. Installation was successful with out any interruption. But after logging in with super admin users icons are not visible and not able to travel through the site.

**Comment by kevinpapst** [comment]
KImai cannot be used in a sub-directory.

**Reply by VasuTech1** [reply]
Kimai is installed in the Public_html/Kimai folder

**Reply by VasuTech1** [reply]
Performed initial setup using this link https://www.kimai.org/documentation/installation.html

**Reply by kevinpapst** [reply]
Where it clearly says: 
<img width="987" alt="Bildschirmfoto 2023-10-12 um 17 56 28" src="https://github.com/kimai/kimai/assets/533162/77821120-f473-47f2-9aa8-60483ff9c287">
